js/sound.js

```diff
@@ -148,6 +148,8 @@ function startBackgroundAudio() {
     updateLegalBackground();
     console.log(backgroundPlay);
     backgroundPlay.play();
+    $(".fa.fa-play").css("color","#0000FF");
+    $(".fa.fa-pause").css("color","#FFFFFF");
```

Comment:
Would it be ok for you to add this to the css files? 
This makes it easier to understand where the change is coming from.

Reply:
@niccokunzmann If we add this to css than it will not work properly like color on play will not be there when we load the page for first time and also I tried it through css and it was overwritten by the materialised.min.css which is hard to change.